content/en/entities/Notification.md

```diff
@@ -81,7 +81,7 @@ aliases: [
 **Version history:**\
 4.0.0 - added
 
-### `relationship_severance_event` {{%optional%}} {#relationship_severance_event}
+### `event` {{%optional%}} {#relationship_severance_event}
```

Comment:
change the `{#anchor}` too?

Reply:
We could, but I was thinking we could keep it for the old changelogs that refer to it.

Reply:
Wouldn't the changelogs need to be fixed, then? They refer to `relationship_severance_event` which is incorrect because it was renamed before official release.

Reply:
Yes, we would also change the changelogs/release notes. But any checked-out copy would still link to the old anchor.

Reply:
Change lgtm.